lib/api/objectPutTagging.js

```diff
@@ -78,10 +78,11 @@ function objectPutTagging(authInfo, request, log, callback) {
             const params = objectMD.versionId ? { versionId:
               objectMD.versionId } : {};
             const replicationInfo =
-                getReplicationInfo(objectKey, bucket, true);
+                getReplicationInfo(objectKey, bucket, true, 0, 'PUT_TAGGING');
```

Comment:
Please create a constant after the requires, this way we don't have to hunt for these constants to maintain them.
```js
const REPLICATION_ACTION = 'PUT_TAGGING';
```

Reply:
Updated.